_includes/footer.html

```diff
@@ -27,18 +52,18 @@ <h2 class="section-title">Legal</h2>
           <li><a href="https://www.eclipse.org/legal/">Legal Resources </a></li>
         </ul>
       </section>
-      <section class="col-sm-6 hidden-print" id="footer-useful-links">
+      <section class="col-sm-5th padding-top-20" id="footer-useful-links">
         <h2 class="section-title">Useful Links</h2>
         <ul class="nav">
-          <li><a href="https://bugs.eclipse.org/bugs/">Report a Bug</a></li>
-          <li><a href="//help.eclipse.org/">Documentation</a></li>
-          <li><a href="https://www.eclipse.org/contribute/">How to Contribute</a></li>
-          <li><a href="https://www.eclipse.org/mail/">Mailing Lists</a></li>
-          <li><a href="https://www.eclipse.org/forums/">Forums</a></li>
-          <li><a href="//marketplace.eclipse.org">Marketplace</a></li>
+          <li><a href="https://github.com/jakartaee/jakarta.ee">Report a Bug</a></li>
+          <li><a href="/resources/">Documentation</a></li>
+          <li><a href="/community/get-involved/">How to Contribute</a></li>
+          <li><a href="/connect/mailing-lists/">Mailing Lists</a></li>
+          <li><a href="/connect/">Forums</a></li>
+          <li><a href="/compatibility/">Marketplace</a></li>
```

Comment:
This does not look right to me. Why is a link to /compatibility has the text Marketplace

I think this is also a bug on the main site.